app/assets/stylesheets/spotlight/_header.scss

```diff
@@ -125,49 +154,59 @@ $menu-link-background-color-hover: rgba(255,255,255,0.15);
   }
 
   &.with-image {
-    .site-title, .search-title {
+    .site-title,
+    .search-title {
       color: $white;
       text-shadow: 1px 1px 0 $black;
     }
+
     small {
       color: $white;
     }
-  }
 
-  .more-exhibits {
-    @extend .navbar-right;
-  }
+    .more-exhibits {
+      a {
+        color: $gray-lighter;
```

Comment:
Properties should be ordered background, border-top-left-radius, border-top-right-radius, color